content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -49,6 +49,17 @@ one if you haven't already.
 1. Install [Kustomize v3.2.1](https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv3.2.1).
 
     Note, Kubeflow is not compatible with later versions of Kustomize. Read [this GitHub issue](https://github.com/kubeflow/manifests/issues/538) for the latest status.
+    
+    To deploy Kustomize v3.2.1 on a Linux box, run the following commands:
+
+    ```
+    curl -LO https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv3.2.1/kustomize_kustomize.v3.2.1_linux_amd64
+    mv kustomize_kustomize.v3.2.1_linux_amd64 kustomize
+    chmod +x ./kustomize
+    sudo mv ./kustomize /usr/local/bin/kustomize
```

Comment:
Consider removing this line, adding a comment to indicate that it is optional, or adding it as an optional step. Users may need to have different versions of Kustomize (since we are asking them to use something other than current version), and adding moving Kustomize into the search path may cause unexpected behaviors.

Reply:
Well, the docs say that Kustomize v3.2.1 is recommended. However, I see your point -- users may have earlier versions of Kustomize or their own Kustomize setup altogether.

I've added the following text:

` # These steps are optional`

Reply:
Hi @jmdevcs and @joeliedtke, I think I need to clarify sth here:
* Kustomize v3.2.1 is not optional for deploying Kubeflow 1.1. The version range that supports Kubeflow is very strict, if you use kustomize v2.x or early v3.x some new features required by Kubeflow haven't been there. Also, if you use Kustomize v3.3+, a breaking change prevents everything from working.
* I believe what @joeliedtke meant was that `sudo mv ./kustomize /usr/local/bin/kustomize` this single line is optional, which I believe is not true either. In Kubeflow on GCP deploy scripts, it calls `kustomize` command, so we must make sure `kustomize` command is in `$PATH`.

My suggestions:
1. Remove the optional statement
2. Add a comment before `sudo mv ./kustomize /usr/local/bin/kustomize`, saying we need to add the `kustomize` tool in your `$PATH`, but you may choose other ways to do so (e.g. change $PATH variable).

Reply:
Hi @Bobgy 

Thank you for the clarification! That was my initial understanding as well.

I have made the recommended suggestions @Bobgy